# HG changeset patch
# User Rik <rik@octave.org>
# Date 1427772671 25200
#      Mon Mar 30 20:31:11 2015 -0700
# Node ID abf348f41759a47c6b2c829281ba0f9870fbfa04
# Parent  10392528396cb05fdd4fde7dcafb651379dac465
es_ES.ts: Update Spanish translation file.

* contributors.in: Add José Luis García Pallero to list of contributors.

* es_ES.ts: Update Spanish translation file (José Luis García Pallero).

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -234,16 +234,17 @@ Peter O'Gorman
 Thorsten Ohl
 Kai T. Ohlhus
 Arno Onken
 Valentin Ortega-Clavero
 Luis F. Ortiz
 Carl Osterwisch
 Janne Olavi Paanajärvi
 Scott Pakin
+José Luis García Pallero
 Jason Alan Palmer
 Gabriele Pannocchia
 Sylvain Pelissier
 Rolando Pereira
 Per Persson
 Primozz Peterlin
 Jim Peterson
 Danilo Piazzalunga
diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -44,17 +44,18 @@
     <message>
         <location filename="../qterminal/libqterminal/QTerminal.cc" line="+65"/>
         <source>foreground</source>
         <translation>primer plano</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>background</source>
-        <translation>plano de fondo</translation>
+        <translation>segundo plano</translation>
+        <translatorcomment>La interpretación correcta en castellano de background es 'segundo plano', en lugar del literal 'plano de fondo'</translatorcomment>
     </message>
     <message>
         <location line="+1"/>
         <source>selection</source>
         <translation>selección</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -77,33 +78,33 @@
     <message>
         <location line="+5"/>
         <source>Select All</source>
         <translation>Seleccionar todo</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Clear Window</source>
-        <translation type="unfinished"></translation>
+        <translation>Limpiar ventana</translation>
     </message>
 </context>
 <context>
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1776"/>
         <source>copied selection to clipboard</source>
         <translation>selección copiada al portapapeles</translation>
     </message>
 </context>
 <context>
     <name>QtHandles::Figure</name>
     <message>
         <location filename="../graphics/Figure.cc" line="+268"/>
         <source>Figure ToolBar</source>
-        <translation type="unfinished">Barra de herramientas para figura</translation>
+        <translation>Barra de herramientas de figura</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>&amp;File</source>
         <translation>&amp;Archivo</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -171,27 +172,27 @@
     <message>
         <location filename="../graphics/MouseModeActionGroup.cc" line="+40"/>
         <source>Rotate</source>
         <translation>Rotar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom In</source>
-        <translation type="unfinished">Aumentar zoom</translation>
+        <translation>Ampliar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Zoom Out</source>
-        <translation type="unfinished">Disminuir zoom</translation>
+        <translation>Reducir</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Pan</source>
-        <translation type="unfinished"></translation>
+        <translation>Mover</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Insert Text</source>
         <translation>Insertar texto</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -231,18 +232,17 @@
     </message>
     <message>
         <location line="-40"/>
         <source>New Function</source>
         <translation>Nueva función</translation>
     </message>
     <message>
         <location line="+1"/>
-        <source>New function name:
-</source>
+        <source>New function name:</source>
         <translation>Nombre de la nueva función:</translation>
     </message>
     <message>
         <location line="+717"/>
         <source>&amp;%1 %2</source>
         <translation>&amp;%1 %2</translation>
     </message>
     <message>
@@ -308,17 +308,17 @@
     <message>
         <location line="+54"/>
         <source>&amp;Next Breakpoint</source>
         <translation>&amp;Siguiente punto de interrupción</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Pre&amp;vious Breakpoint</source>
-        <translation>Punto de interrupción &amp;previo</translation>
+        <translation>Punto de interrupción &amp;anterior</translation>
     </message>
     <message>
         <location line="-6"/>
         <source>Toggle &amp;Breakpoint</source>
         <translation>&amp;Alternar puntos de interrupción</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -357,19 +357,18 @@
     </message>
     <message>
         <location line="-90"/>
         <source>Go &amp;to Line...</source>
         <translation>&amp;Ir a línea...</translation>
     </message>
     <message>
         <location line="-1126"/>
-        <source>Could not start custom file editor
-%1</source>
-        <translation type="unfinished">No se ha podido iniciar el editor de archivos personalizado %1</translation>
+        <source>Could not start custom file editor %1</source>
+        <translation>No se ha podido iniciar el editor de archivos personalizado %1</translation>
     </message>
     <message>
         <location line="+148"/>
         <source>Create</source>
         <translation>Crear</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -449,37 +448,37 @@
     <message>
         <location line="+5"/>
         <source>Duplicate Selection/Line</source>
         <translation>Duplicar selección/línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
-        <translation>Transponer línea</translation>
+        <translation>Intercambiar con línea anterior</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Show Completion List</source>
-        <translation type="unfinished">&amp;Mostrar lista de terminación</translation>
+        <translation>&amp;Mostrar lista de comandos coincidentes</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Format</source>
         <translation>&amp;Formato</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Uppercase Selection</source>
-        <translation type="unfinished">Seleccionar &amp;mayusculas</translation>
+        <translation>Convertir selección a &amp;mayúsculas</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Lowercase Selection</source>
-        <translation type="unfinished">Seleccionar m&amp;inúsculas</translation>
+        <translation>Convertir selección a m&amp;inúsculas</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>Convert Line Endings to &amp;Windows (CRLF)</source>
         <translation>Convertir los finales de línea a &amp;Windows (CRLF)</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -494,23 +493,23 @@
     <message>
         <location line="+3"/>
         <source>Navi&amp;gation</source>
         <translation>&amp;Navegación</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Move to Matching Brace</source>
-        <translatorcomment>Llave o paréntesis en español.</translatorcomment>
-        <translation>Mover a la llave correspondiente</translation>
+        <translation>Mover al paréntesis/corchete/llave correspondiente</translation>
+        <translatorcomment>Las posibilidades son aréntesis, corchete o llave.</translatorcomment>
     </message>
     <message>
         <location line="+2"/>
         <source>Select to Matching Brace</source>
-        <translation type="unfinished">Seleccionar hasta la llave correspondiente</translation>
+        <translation>Seleccionar hasta el paréntesis/corchete/llave correspondiente</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>&amp;Preferences...</source>
         <translation>&amp;Preferencias...</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -525,17 +524,17 @@
     <message>
         <location line="+2"/>
         <source>&amp;Editor</source>
         <translation>&amp;Editor</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show &amp;Line Numbers</source>
-        <translation>Mostrar número de &amp;línea</translation>
+        <translation>Mostrar números de &amp;línea</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;White Spaces</source>
         <translation>Mostrar espacios en &amp;blanco</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -545,29 +544,27 @@
     <message>
         <location line="+4"/>
         <source>Show &amp;Indentation Guides</source>
         <translation>Mostrar guías de &amp;sangría</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Long Line &amp;Marker</source>
-        <translation>Mostrar &amp;marcador largo de línea</translation>
+        <translation>Mostrar &amp;marcador de longitud de línea</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Zoom &amp;In</source>
-        <translatorcomment>the term zoom is not well defined in spanish</translatorcomment>
-        <translation type="unfinished">&amp;Aumentar zoom</translation>
+        <translation>&amp;Ampliar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom &amp;Out</source>
-        <translatorcomment>the term zoom is not well defined in spanish</translatorcomment>
-        <translation type="unfinished">&amp;Disminuir zoom</translation>
+        <translation>&amp;Disminuir</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Normal Size</source>
         <translation>Tamaño &amp;normal</translation>
     </message>
     <message>
         <location line="+32"/>
@@ -586,40 +583,32 @@
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Documentation on Keyword</source>
         <translation>&amp;Documentación de palabras clave</translation>
     </message>
     <message>
         <location line="-1103"/>
-        <source>Could not open file
-%1
-for read: %2.</source>
+        <source>Could not open file\n%1\nfor read: %2.</source>
         <translation>No se ha podido abrir el archivo\n%1\npara lectura: %2.</translation>
     </message>
     <message>
         <location line="+19"/>
-        <source>File
-%1
-does not exist. Do you want to create it?</source>
+        <source>File\n%1\ndoes not exist. Do you want to create it?</source>
         <translation>El archivo\n%1\nno existe. ¿Desea crearlo?</translation>
     </message>
     <message>
         <location line="+25"/>
-        <source>Could not open file
-%1
-for write: %2.</source>
+        <source>Could not open file\n%1\nfor write: %2.</source>
         <translation>No se ha podido abrir el archivo\n%1\npara escritura: %2.</translation>
     </message>
     <message>
         <location line="+62"/>
-        <source>File not saved! A file with the selected name
-%1
-is already open in the editor</source>
+        <source>File not saved! A file with the selected name\n%1\nis already open in the editor</source>
         <translation>¡Archivo no guardado! Un archivo con el nombre seleccionado\n%1\nya se encuentra abierto en el editor</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>The associated file editor tab has disappeared.</source>
         <translation>La pestaña para la edición del archivo ha desaparecido.</translation>
     </message>
     <message>
@@ -643,72 +632,63 @@ is already open in the editor</source>
         <translation>&amp;Ejecutar</translation>
     </message>
 </context>
 <context>
     <name>file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+92"/>
         <source>eol:</source>
-        <translatorcomment>It may be a reference to end-of-line (&quot;fin de línea&quot; in spanish)</translatorcomment>
-        <translation type="unfinished">eol:</translation>
+        <translation>Fin de línea:</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>line:</source>
         <translation>Línea:</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>col:</source>
         <translation>Col:</translation>
     </message>
     <message>
         <location line="+156"/>
         <source>%1 is a built-in function</source>
-        <translation type="unfinished">%1 es una función incorporada</translation>
+        <translation>%1 es una función integrada en Octave</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Can not find function %1</source>
         <translation>No se encuentra la función %1</translation>
     </message>
     <message>
         <location line="+775"/>
         <source>Goto line</source>
-        <translation>Ir a línea</translation>
+        <translation>Ir a la línea</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Line number</source>
         <translation>Número de línea</translation>
     </message>
     <message>
         <location line="+146"/>
         <location line="+63"/>
         <source>&lt;unnamed&gt;</source>
         <translation>&lt;sin nombre&gt;</translation>
     </message>
     <message>
         <location line="+4"/>
-        <source>The file
-
-  %1
-
-is about to be closed but has been modified.  %2</source>
-        <translation>El archivo
-
-  %1
-
-va a ser cerrado pero ha sido previamente modificado.  %2</translation>
+        <source>The file\n\n%1\n\nis about to be closed but has been modified.  %2</source>
+        <translation>El archivo\n\n%1\n\nva a ser cerrado, pero ha sido modificado.  %2</translation>
     </message>
     <message>
         <location line="+288"/>
         <source>Line Endings:</source>
-        <translation>Finales de línea:</translation>
+        <translation>Fin de línea:</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Octave Files (*.m)</source>
         <translation>Archivos de Octave (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -727,54 +707,38 @@ va a ser cerrado pero ha sido previament
         <location line="+177"/>
         <location line="+85"/>
         <location line="+23"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
         <location line="-6"/>
-        <source>
-
-Warning: The contents in the editor is modified!</source>
-        <translation>\n\n Advertencia: ¡El contenido del editor ha sido modificado!</translation>
+        <source>\n\nWarning: The contents in the editor is modified!</source>
+        <translation>\n\nAdvertencia: ¡el contenido del editor ha sido modificado!</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>It seems that the file
-%1
-has been deleted or renamed. Do you want to save it now?%2</source>
-        <translation>Parece que el archivo\n%1\nha sido eliminado o renombrado.¿Desea guardarlo ahora?%2</translation>
+        <source>It seems that the file\n%1\nhas been deleted or renamed. Do you want to save it now?%2</source>
+        <translation>Parece que el archivo\n%1\nha sido eliminado o renombrado. ¿Desea guardarlo ahora?%2</translation>
     </message>
     <message>
         <location line="-285"/>
-        <source>Could not open file %1 for write:
-%2.</source>
+        <source>Could not open file %1 for write:\n%2.</source>
         <translation>No se ha podido abrir el archivo %1 para escritura:\n%2.</translation>
     </message>
     <message>
         <location line="+177"/>
-        <source>&quot;%1&quot;
-is not a valid identifier.
-
-If you keep this file name, you will not be able to
-call your script using its name as an Octave command.
-
-Do you want to choose another name?</source>
-        <translation>&quot;%1&quot;
-no es un identificador válido.
-
-Si se mantiene este nombre de archivo, no será posible llamar al script utilizando su nombre como un comando Octave.
-
-¿Desea elegir otro nombre?</translation>
+        <source>&quot;%1&quot;\nis not a valid identifier.\n\nIf you keep this file name, you will not be able to call your script using its name as an Octave command.\n\nDo you want to choose another name?</source>
+        <translation>&quot;%1&quot;\nno es un identificador válido.\n\nSi se mantiene este nombre de archivo, no será posible llamar al script utilizando su nombre como un comando de Octave.\n\n¿Desea elegir otro nombre?</translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
-        <translation>Parece que el archivo \&apos;%1\&apos; ha sido modificado por otra aplicación. ¿Desea recargarlo?</translation>
+        <translation>Parece que el archivo &apos;%1&apos; ha sido modificado por otra aplicación. ¿Desea recargarlo?</translation>
     </message>
 </context>
 <context>
     <name>files_dock_widget</name>
     <message>
         <location filename="../src/files-dock-widget.cc" line="+68"/>
         <source>File Browser</source>
         <translation>Explorador de archivos</translation>
@@ -813,32 +777,32 @@ Si se mantiene este nombre de archivo, no será posible llamar al script utilizando su nombre como un comando Octave.
     <message>
         <location line="-783"/>
         <source>Show Octave directory</source>
         <translation>Mostrar el directorio de Octave</translation>
     </message>
     <message>
         <location line="-4"/>
         <source>One directory up</source>
-        <translation type="unfinished">Directorio superior</translation>
+        <translation>Directorio superior</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Go to current Octave directory</source>
         <translation>Ir al directorio actual de Octave</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set Octave directory</source>
         <translation>Seleccionar el directorio de Octave</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Set Octave directroy to current browser directory</source>
-        <translation>Asignar el directorio de Octave como directorio actual del explorador</translation>
+        <translation>Asignar el directorio actual del explorador como directorio de Octave</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Show Home Directory</source>
         <translation>Mostrar el directorio de inicio</translation>
     </message>
     <message>
         <location line="+20"/>
@@ -850,17 +814,17 @@ Si se mantiene este nombre de archivo, no será posible llamar al script utilizando su nombre como un comando Octave.
         <location line="-394"/>
         <location line="+397"/>
         <source>New Directory...</source>
         <translation>Nuevo directorio...</translation>
     </message>
     <message>
         <location line="-335"/>
         <source>Activate to open in editor, right click for alternatives</source>
-        <translation>Activar para abrir en editor, presione clic derecho para alternativas</translation>
+        <translation>Activar para abrir en el editor, haga clic derecho para ver las alternativas</translation>
     </message>
     <message>
         <location line="+236"/>
         <source>File size</source>
         <translation>Tamaño del archivo</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -919,36 +883,33 @@ Si se mantiene este nombre de archivo, no será posible llamar al script utilizando su nombre como un comando Octave.
     </message>
     <message>
         <location line="+115"/>
         <source>Rename file/directory</source>
         <translation>Renombrar archivo/directorio</translation>
     </message>
     <message>
         <location line="+1"/>
-        <source>Rename file/directory:
-</source>
+        <source>Rename file/directory:</source>
         <translation>Renombrar archivo/directorio:</translation>
     </message>
     <message>
         <location line="+1"/>
-        <source>
- to: </source>
+        <source>\nto: </source>
         <translation>\na: </translation>
     </message>
     <message>
         <location line="+25"/>
         <location line="+14"/>
         <source>Delete file/directory</source>
         <translation>Eliminar archivo/directorio</translation>
     </message>
     <message>
         <location line="-13"/>
-        <source>Are you sure you want to delete
-</source>
+        <source>Are you sure you want to delete</source>
         <translation>¿Está seguro de que desea eliminar</translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Can not delete a directory that is not empty</source>
         <translation>No se puede eliminar un directorio que no esté vacío</translation>
     </message>
     <message>
@@ -988,22 +949,22 @@ Si se mantiene este nombre de archivo, no será posible llamar al script utilizando su nombre como un comando Octave.
 </translation>
     </message>
 </context>
 <context>
     <name>final_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+199"/>
         <source>Enjoy!</source>
-        <translation>¡Que lo disfrute!</translation>
+        <translation>¡Disfrútelo!</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Previous</source>
-        <translation>Previo</translation>
+        <translation>Anterior</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Finish</source>
         <translation>Terminar</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1167,97 +1128,97 @@ a:link { text-decoration: underline; col
     <message>
         <location filename="../src/find-files-dialog.cc" line="+52"/>
         <source>Find Files</source>
         <translation>Buscar archivos</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Named:</source>
-        <translation>Nombrado:</translation>
+        <translation>Denominado:</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Start in:</source>
-        <translation>Iniciar en:</translation>
+        <translation>Buscar en:</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Enter the start directory</source>
-        <translation>Introducir directorio de inicio</translation>
+        <translation>Introduzca el directorio superior de búsqueda</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Browse...</source>
         <translation>Explorar...</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Browse for start directory</source>
-        <translation>Explorar para seleccionar el directorio de inicio</translation>
+        <translation>Explorar para seleccionar el directorio superior de búsqueda</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Search recursively through directories for matching files</source>
         <translation>Buscar archivos de forma recursiva en los directorios</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Include matching directories in search results</source>
         <translation>Incluir directorios que coicidan en los resultados de la búsqueda</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Name case insensitive</source>
-        <translation>Distinguir mayúsculas/minúsculas en nombre</translation>
+        <translation>No distinguir mayúsculas de minúsculas</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Set matching name is case insensitive</source>
-        <translation>Establecer distinción entre mayúsculas y minúsculas en el nombre</translation>
+        <translation>Establecer la no distinción entre mayúsculas y minúsculas</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Contains text:</source>
-        <translation>Contiene el texto:</translation>
+        <translation>Conteniendo el texto:</translation>
     </message>
     <message>
         <location line="-35"/>
         <source>Enter the filename search expression</source>
-        <translation type="unfinished">Introduzca expresión para busqueda del archivo</translation>
+        <translation>Introduzca expresión de búsqueda para los nombres de archivo</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Search subdirectories</source>
-        <translation>Buscar subdirectorios</translation>
+        <translation>Buscar en subdirectorios</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Include directory names</source>
         <translation>Incluir nombres de directorios</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Enter the file content search expression</source>
-        <translation type="unfinished">Introducir expresion de busqueda con contenido del archivo</translation>
+        <translation>Busca en el contenido del fichero</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Text to match</source>
-        <translation>Texto a coincidir</translation>
+        <translation>Texto a buscar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Text case insensitive</source>
-        <translation>Distinguir mayúsculas/minúsculas en texto</translation>
+        <translation>No distinguir entre mayúsculas y minúsculas</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Set text content is case insensitive</source>
-        <translation>Establecer distinción entre mayúsculas y minúsculas en el texto</translation>
+        <translation>Establecer la no distinción entre mayúsculas y minúsculas</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Search results</source>
         <translation>Resultados de la búsqueda</translation>
     </message>
     <message>
         <location line="+17"/>
@@ -1267,27 +1228,27 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Find</source>
         <translation>Buscar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Start search for matching files</source>
-        <translation>Empezar búsqueda de archivos coincidentes</translation>
+        <translation>Empezar la búsqueda</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Stop</source>
         <translation>Detener</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Stop searching</source>
-        <translation>Detener búsqueda</translation>
+        <translation>Detener la búsqueda</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>File name/location</source>
         <translation>Nombre de archivo/ubicación</translation>
     </message>
     <message>
         <location line="+17"/>
@@ -1297,22 +1258,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+105"/>
         <source>Searching...</source>
         <translation>Buscando...</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>%1 match(es)</source>
-        <translation type="unfinished">%1 resultado(s) coincidente(s)</translation>
+        <translation>%1 resultado(s) coincidente(s)</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Set search directory</source>
-        <translation>Seleccionar directorio de búsqueda</translation>
+        <translation>Seleccionar el directorio de búsqueda</translation>
     </message>
 </context>
 <context>
     <name>find_files_model</name>
     <message>
         <location filename="../src/find-files-model.cc" line="+82"/>
         <source>Filename</source>
         <translation>Nombre de archivo</translation>
@@ -1328,17 +1289,17 @@ a:link { text-decoration: underline; col
     <message>
         <location filename="../src/history-dock-widget.cc" line="+47"/>
         <source>Browse and search the command history.</source>
         <translation>Explorar y buscar en el historial de comandos.</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Double-click a command to transfer it to the terminal.</source>
-        <translation>Pulse dos veces un comando para transferirlo al terminal.</translation>
+        <translation>Haga doble clic en un comando para transferirlo al terminal.</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Enter text to filter the command history</source>
         <translation>Introduzca texto para filtrar el historial de comandos</translation>
     </message>
     <message>
         <location line="+10"/>
@@ -1387,28 +1348,28 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;You seem to be using the Octave graphical interface for the first time on this computer.
 Click &apos;Next&apos; to create a configuration file and launch Octave.&lt;/p&gt;
 &lt;p&gt;The configuration file is stored in&lt;br&gt;%1.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
-&lt;p&gt;Parece que está utilizando la interface gráfica de Octave por primera vez en esta computadora.
+&lt;p&gt;Parece que está utilizando la interfaz gráfica de Octave por primera vez en esta computadora.
 Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
 &lt;p&gt;El archivo de configuración se guardará en&lt;br&gt;%1.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
         <location filename="../src/main-window.cc" line="+262"/>
         <source>Load Workspace</source>
-        <translation>Cargar espacio de trabajo</translation>
+        <translation>Cargar el espacio de trabajo</translation>
     </message>
     <message>
         <location line="+726"/>
         <location line="+917"/>
         <source>About Octave</source>
         <translation>Acerca de Octave</translation>
     </message>
     <message>
@@ -1454,42 +1415,42 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     <message>
         <location line="+4"/>
         <source>Paste</source>
         <translation>Pegar</translation>
     </message>
     <message>
         <location line="-1458"/>
         <source>Save Workspace As</source>
-        <translation>Guardar espacio de trabajo como</translation>
+        <translation>Guardar el espacio de trabajo como</translation>
     </message>
     <message>
         <location line="+131"/>
         <source>The release notes file &apos;%1&apos; is empty.</source>
-        <translation>El archivo de las notas de versión %1 está vacío.</translation>
+        <translation>El archivo de notas de versión %1 está vacío.</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>The release notes file &apos;%1&apos; cannot be read.</source>
-        <translation>El archivo de las notas de versión %1 no puede ser leído.</translation>
+        <translation>El archivo de notas de versión %1 no puede ser leído.</translation>
     </message>
     <message>
         <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Notas de la versión de Octave</translation>
     </message>
     <message>
         <location line="+192"/>
         <source>Octave Community News</source>
         <translation>Noticas de la comunidad Octave</translation>
     </message>
     <message>
         <location line="+185"/>
         <source>Are you sure you want to exit Octave?</source>
-        <translation>¿Está seguro que desea salir de Octave?</translation>
+        <translation>¿Está seguro de que desea salir de Octave?</translation>
     </message>
     <message>
         <location line="+839"/>
         <source>Open an existing file in editor</source>
         <translation>Abrir un archivo existente en el editor</translation>
     </message>
     <message>
         <location line="+57"/>
@@ -1569,47 +1530,47 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     <message>
         <location line="+853"/>
         <source>Load Workspace...</source>
         <translation>Cargar espacio de trabajo...</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save Workspace As...</source>
-        <translation>Guardar espacio de trabajo como...</translation>
+        <translation>Guardar el espacio de trabajo como...</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>New Script</source>
-        <translation type="unfinished">Nuevo script (guión)</translation>
+        <translation>Nuevo guión (script)</translation>
     </message>
     <message>
         <location line="+115"/>
         <source>Step Out</source>
         <translation>Terminar</translation>
     </message>
     <message>
         <location line="+94"/>
         <source>Reset Default Window Layout</source>
-        <translation>Reestablecer el esquema de ventana predeterminado</translation>
+        <translation>Restablecer el esquema de ventana predeterminado</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>Octave Packages</source>
         <translation>Paquetes de Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Share Code</source>
         <translation>Compartir código</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Contribute to Octave</source>
-        <translation>Colaborar en Octave</translation>
+        <translation>Colaborar con Octave</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Developer Resources</source>
         <translation>Recursos para el desarrollador de Octave</translation>
     </message>
     <message>
         <location line="+14"/>
@@ -1655,17 +1616,17 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
         <location line="-1135"/>
         <location line="+1138"/>
         <source>Browse directories</source>
         <translation>Explorar directorios</translation>
     </message>
     <message>
         <location line="-176"/>
         <source>Quit Debug Mode</source>
-        <translation>Salir de modo de depuración</translation>
+        <translation>Salir del modo de depuración</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>&amp;Window</source>
         <translation>&amp;Ventana</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1746,17 +1707,17 @@ when you have a connection to the web (l
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; The Octave Developers, </source>
         <translation>&lt;html&gt;
 &lt;body&gt;
 &lt;p&gt;
 Las fuentes de noticias de la comunidad Octave parece que no están disponibles.
 &lt;/p&gt;
 &lt;p&gt;
-Para las últimas novedades, visite
+Para ver las últimas novedades, visite
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
 cuando tenga conexión a la web (el enlace se abre en un navegador externo).
 &lt;/p&gt;
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave,</translation>
     </message>
     <message>
         <location line="+18"/>
@@ -1771,23 +1732,23 @@ For the latest news, please check
 when you have a connection to the web (link opens in an external browser)
 or enable web connections for news in Octave&apos;s network settings dialog.
 &lt;/p&gt;
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; The Octave Developers, </source>
         <translation>&lt;html&gt;
 &lt;body&gt;
 &lt;p&gt;
-La conexión a la web para desplegar las últimas novedades sobre Octave ha sido desactivada.
+La conexión a la web para mostrar las últimas novedades sobre Octave ha sido desactivada.
 &lt;/p&gt;
 &lt;p&gt;
-Para ver las últimas novedades, vea
+Para ver las últimas novedades, vaya a
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;
 cuando tenga conexión a la web (el enlace se abre en un navegador externo)
-o active las conexiones de web para ver las novedades en el diálogo de configuración de red de Octave
+o active la conexión a la web para ver las novedades en el diálogo de configuración de red de Octave
 &lt;/p&gt;
 &lt;p&gt;
 &lt;small&gt;&lt;em&gt;&amp;mdash; Los desarrolladores de Octave,
 </translation>
     </message>
 </context>
 <context>
     <name>octave_dock_widget</name>
@@ -1835,80 +1796,71 @@ o active las conexiones de web para ver las novedades en el diálogo de configuración de red de Octave
         <location filename="../src/octave-qt-link.cc" line="+111"/>
         <location line="+6"/>
         <location line="+7"/>
         <source>Create</source>
         <translation>Crear</translation>
     </message>
     <message>
         <location line="-10"/>
-        <source>File
-%1
-does not exist. Do you want to create it?</source>
+        <source>File\n%1\ndoes not exist. Do you want to create it?</source>
         <translation>El archivo\n%1\nno existe. ¿Desea crearlo?</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
         <location line="+211"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation type="unfinished">El archivo %1 no existe en la ruta de carga.  Para correr o depurar la función que está editando, tiene que cambiar al directorio %2 o agregar ese directorio a la ruta de carga.</translation>
+        <translation>El archivo %1 no existe en las rutas de búsqueda de funciones (path). Para ejecutar o depurar la función que está editando, tiene que cambiar al directorio %2 o agregar ese directorio a las rutas de búsqueda de funciones (path).</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
-        <translation type="unfinished">El archivo %1 está ensombrecido por un archivo con el mismo nombre en la ruta de carga. Para ejecutar o depurar la función que se está editando, cambie al directorio %2.</translation>
+        <translation>Existe un archivo con el mismo nombre que %1 en la ruta de búsqueda de funciones (path). Para ejecutar o depurar la función que se está editando, cambie al directorio %2.</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory or Add Directory to Load Path</source>
-        <translation>Cambiar directorio o agregar directorio a la ruta de carga</translation>
+        <translation>Cambiar directorio o agregar directorio a la ruta de búsqueda de funciones (path)</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Change Directory</source>
         <translation>Cambiar directorio</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Add Directory to Load Path</source>
         <translatorcomment>Uso la traducción &quot;ruta&quot; para &quot;path&quot;: http://es.wikipedia.org/wiki/Ruta_(informática)</translatorcomment>
-        <translation>Agregar directorio a la ruta de carga</translation>
+        <translation>Agregar directorio a la ruta de búsqueda de funciones (path)</translation>
     </message>
     <message>
         <location line="-223"/>
         <location line="+224"/>
         <source>Cancel</source>
         <translation>Cancelar</translation>
     </message>
 </context>
 <context>
     <name>octave_txt_lexer</name>
     <message>
         <location filename="../src/m-editor/octave-txt-lexer.cc" line="+39"/>
         <source>Default</source>
-        <translation type="unfinished">Por defecto</translation>
+        <translation>Por defecto</translation>
     </message>
 </context>
 <context>
     <name>resource_manager</name>
     <message>
         <location filename="../src/resource-manager.cc" line="+245"/>
-        <source>The settings file
-%1
-does not exist and can not be created.
-Make sure you have read and write permissions to
-%2
-
-Octave GUI must be closed now.</source>
-        <translation>El archivo de configuración\n%1\nno existe y no puede ser creado.
-Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
+        <source>The settings file\n%1\ndoes not exist and can not be created.\nMake sure you have read and write permissions to\n%2\n\nOctave GUI must be closed now.</source>
+        <translation>El archivo de configuración\n%1\nno existe y no se puede crear.\nAsegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave se cerrará.</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Octave Critical Error</source>
         <translation>Error crítico de Octave</translation>
     </message>
 </context>
 <context>
@@ -1927,82 +1879,82 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="-338"/>
         <source>Octave logo only</source>
         <translation>Sólo logo de Octave</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Letter icons</source>
-        <translation>Iconos de letras</translation>
+        <translation>Iconos de texto</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Graphic icons</source>
         <translation>Iconos gráficos</translation>
     </message>
     <message>
         <location line="-118"/>
         <source>Dock widget title bar</source>
-        <translation type="unfinished">Acoplar barra de título de widget</translation>
+        <translation>Acoplar widget de barra de título</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Small</source>
-        <translation type="unfinished"></translation>
+        <translation>Pequeño</translation>
     </message>
     <message>
         <location line="+7"/>
         <source> Normal</source>
-        <translation type="unfinished"></translation>
+        <translation>Normal</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Large</source>
-        <translation type="unfinished"></translation>
+        <translation>Grande</translation>
     </message>
     <message>
         <location line="+168"/>
         <source>Custom style</source>
         <translation>Estilo personalizado</translation>
     </message>
     <message>
         <location line="+92"/>
         <source>These preferences are applied after any .octaverc startup files.</source>
-        <translation type="unfinished"></translation>
+        <translation>Estas configuraciones se aplicarán después de la ejecución de los ficheros de inicio .octaverc</translation>
     </message>
     <message>
         <location line="+55"/>
         <source>Restore working directory of previous session</source>
-        <translation type="unfinished"></translation>
+        <translation>Restaurar el directorio de trabajo de la sesión anterior</translation>
     </message>
     <message>
         <location line="+33"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
     <message>
         <location line="+151"/>
         <source>Color</source>
         <translation>Color</translation>
     </message>
     <message>
         <location line="+138"/>
         <source>Draw a long line marker</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar marcador de longitud de línea</translation>
     </message>
     <message>
         <location line="+48"/>
         <source>Show EOL characters</source>
-        <translation type="unfinished"></translation>
+        <translation>Mostrar caracteres de fin de línea</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Default EOL mode</source>
-        <translation type="unfinished"></translation>
+        <translation>Identificador de fin de línea por defecto</translation>
     </message>
     <message>
         <location line="+154"/>
         <source>Indent width</source>
         <translation>Ancho de sangría</translation>
     </message>
     <message>
         <location line="+7"/>
@@ -2047,17 +1999,17 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="-10"/>
         <source>Match words in document</source>
         <translation>Coincidir palabras en documento</translation>
     </message>
     <message>
         <location line="+173"/>
         <source>Restore editor tabs from previous session on startup</source>
-        <translation>Restaurar las pestañas del editor de la sesión anterior en el arranque </translation>
+        <translation>En el arranque, restaurar las pestañas del editor de la sesión anterior</translation>
     </message>
     <message>
         <location line="-426"/>
         <source>Use custom file editor</source>
         <translation>Utilizar editor de archivos personalizado</translation>
     </message>
     <message>
         <location line="+482"/>
@@ -2098,17 +2050,17 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="+39"/>
         <source>Do not show whitespace used for indentation</source>
         <translation>No mostrar los espacios usados para la sangría</translation>
     </message>
     <message>
         <location line="+625"/>
         <source>Number of characters before list is shown: </source>
-        <translation type="unfinished"></translation>
+        <translation>Número de caracteres a escribir antes de mostrar la lista</translation>
     </message>
     <message>
         <location line="+112"/>
         <source>Create nonexistent files without prompting</source>
         <translation>Crear archivos no existentes sin consultar</translation>
     </message>
     <message>
         <location line="-429"/>
@@ -2138,17 +2090,17 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="+59"/>
         <source>Font size</source>
         <translation>Tamaño de fuente</translation>
     </message>
     <message>
         <location line="+123"/>
         <source>Set focus to terminal when running a command from within another widget</source>
-        <translation>Aplicar enfoque al terminal cuando se ejecute un comando desde otro widget</translation>
+        <translation>Foco al terminal cuando se ejecute un comando desde otro widget</translation>
     </message>
     <message>
         <location line="+73"/>
         <source>File Browser</source>
         <translation>Explorador de archivos</translation>
     </message>
     <message>
         <location line="+46"/>
@@ -2168,17 +2120,17 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="-14"/>
         <source>Show hidden files</source>
         <translation>Mostrar archivos ocultos</translation>
     </message>
     <message>
         <location line="-1715"/>
         <source>Interface</source>
-        <translation>Interfase</translation>
+        <translation>Interfaz</translation>
     </message>
     <message>
         <location line="+52"/>
         <source>Confirm before exiting</source>
         <translation>Confirmar antes de salir</translation>
     </message>
     <message>
         <location line="+103"/>
@@ -2200,204 +2152,204 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="+39"/>
         <source>Background inactive</source>
         <translation>Fondo inactivo</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>3D</source>
-        <translation type="unfinished"></translation>
+        <translation>3D</translation>
     </message>
     <message>
         <location line="+54"/>
         <source>Octave Startup</source>
         <translation>Inicio de Octave</translation>
     </message>
     <message>
         <location line="+41"/>
         <location line="+1392"/>
         <source>Startup path</source>
-        <translation type="unfinished">Dirección de inicio</translation>
+        <translation>Dirección de inicio</translation>
     </message>
     <message>
         <location line="-1379"/>
         <location line="+1409"/>
         <source>Browse</source>
-        <translation type="unfinished">Explorar</translation>
+        <translation>Explorar</translation>
     </message>
     <message>
         <location line="-1322"/>
         <source>After Column</source>
-        <translation type="unfinished">Despues de columna</translation>
+        <translation>Despues de la columna</translation>
     </message>
     <message>
         <location line="+144"/>
         <source>Tab width min.</source>
-        <translation type="unfinished">Ancho min. de tabulador.</translation>
+        <translation>Anchura mínima del tabulador.</translation>
     </message>
     <message>
         <location line="+64"/>
         <source>max.</source>
-        <translation>max.</translation>
+        <translation>máx.</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>This works well for monospaced fonts. The line is drawn at a position based on the width of a space character in the default font. It may not work very well if styles use proportional fonts or if varied font sizes or bold, italic and normal texts are used.</source>
-        <translation>Esto funciona correctamente con fuentes monoespaciadas. La línea es dibujada en una posición basada en lo ancho de un caracter en la fuente por defecto. Esta podría no funcionar correctamente si los estilos utilizan fuentes proporcionales o si se utilizan fuentes de varios tamaños o negritas, itálicas y textos normales.</translation>
+        <translation>Esto funciona correctamente con fuentes con espaciado fijo. Se dibuja la línea en una posición basada la anchura de un carácter de la fuente por defecto. Esto podría no funcionar correctamente si los estilos utilizan fuentes proporcionales o si se utilizan fuentes de varios tamaños, negrita, cursiva y textos normales.</translation>
     </message>
     <message>
         <location line="+19"/>
         <source>Enable Code Folding</source>
-        <translation type="unfinished">Activar código plegable</translation>
+        <translation>Activar plegado de código</translation>
     </message>
     <message>
         <location line="+61"/>
         <source>Windows (CRLF)</source>
-        <translation type="unfinished">Windows (CRLF)</translation>
+        <translation>Windows (CRLF)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Mac (CR)</source>
-        <translation type="unfinished">Mac (CR)</translation>
+        <translation>Mac (CR)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Unix (LF)</source>
-        <translation type="unfinished">Unix (LF)</translation>
+        <translation>Unix (LF)</translation>
     </message>
     <message>
         <location line="+32"/>
         <source>Show horizontal scroll bar</source>
         <translation>Mostrar barra de desplazamiento horizontal</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Indentation</source>
-        <translation type="unfinished"></translation>
+        <translation>Sangrado</translation>
     </message>
     <message>
         <location line="+117"/>
         <source>Indentation uses tabs</source>
-        <translation type="unfinished">Sangrado por medio de tabulador</translation>
+        <translation>Sangrado con tabulador</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>Auto completion</source>
-        <translation type="unfinished">Terminación automática</translation>
+        <translation>Completado automático</translation>
     </message>
     <message>
         <location line="+101"/>
         <source>With Octave builtins</source>
-        <translation type="unfinished">Con funciones internas de Octave</translation>
+        <translation>Con funciones internas de Octave</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>With Octave functions</source>
         <translation>Con funciones de Octave</translation>
     </message>
     <message>
         <location line="+64"/>
         <source>Show completion list automatically</source>
-        <translation type="unfinished">Mostrar lista de terminación automáticamente</translation>
+        <translation>Mostrar lista de comandos coincidentes automáticamente</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>File handling</source>
-        <translation type="unfinished"></translation>
+        <translation>Manejo de ficheros</translation>
     </message>
     <message>
         <location line="+40"/>
         <source>Reload externally changed files without prompt</source>
-        <translation type="unfinished">Recargar los archivos modificados externamente sin preguntar</translation>
+        <translation>Recargar los archivos modificados externamente sin consultar</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select font, font size (as a difference from the default size), font style (&lt;b&gt;b&lt;/b&gt;old, &lt;b&gt;i&lt;/b&gt;talic, &lt;b&gt;u&lt;/b&gt;nderline), text color, and background color (for the latter, the color magenta (255,0,255) is a placeholder for the default background color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selección de fuente, tamaño (como diferencia con respecto al tamaño por defecto), estilo (&lt;b&gt;n&lt;/b&gt;egrita, &lt;b&gt;c&lt;/b&gt;ursiva, &lt;b&gt;s&lt;/b&gt;ubrayado), color y color de fondo (para el último caso, el magenta (255,0,255) es un sustituto para el color de fondo por defecto).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+192"/>
         <source>(Changing buffer size clears history)</source>
-        <translation>(Cambiar el tamaño del búfer elimina el historial)</translation>
+        <translation>(Cambiar el tamaño del buffer elimina el historial)</translation>
     </message>
     <message>
         <location line="+28"/>
         <source>History buffer Size</source>
-        <translation>Tamaño del búfer de historial</translation>
+        <translation>Tamaño del buffer para el historial</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Print debug location in terminal window in addition to the marker in the editor</source>
-        <translation type="unfinished">Imprimir ubicación de depuración en la ventana del terminal además de en el editor</translation>
+        <translation>Imprimir la ubicación de depuración en la ventana del terminal además de en el editor</translation>
     </message>
     <message>
         <location line="+87"/>
         <source>Display</source>
-        <translation type="unfinished">Mostrar</translation>
+        <translation>Mostrar</translation>
     </message>
     <message>
         <location line="+39"/>
         <source>Alternating row colors</source>
         <translation>Alternar colores de filas</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Behavior</source>
         <translation>Comportamiento</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Restore last directory of previous session</source>
-        <translation>Restaurar último directorio de sesión previa</translation>
+        <translation>Restaurar último directorio de la sesión previa</translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Workspace</source>
         <translation>Espacio de trabajo</translation>
     </message>
     <message>
         <location line="+42"/>
         <source>Hide tool tips</source>
-        <translation type="unfinished"></translation>
+        <translation>Ocultar sugerencias automáticas</translation>
     </message>
     <message>
         <location line="+182"/>
         <source>Edit a shortcut by double-clicking in a cell</source>
-        <translation type="unfinished"></translation>
+        <translation>Los atajos se pueden editar haciendo doble clic en la celda correspondiente</translation>
     </message>
     <message>
         <location line="+108"/>
         <source>Network</source>
         <translation>Red</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Allow Octave to connect to the Octave web site to display current news and information</source>
-        <translation>Permitir a Octave la conexión al sitio web para desplegar las últimas noticias e información</translation>
+        <translation>Permitir a Octave la conexión al sitio web para mostrar las últimas noticias e información</translation>
     </message>
     <message>
         <location line="+46"/>
         <source>Use proxy server</source>
         <translation>Utilizar servidor &quot;proxy&quot;</translation>
     </message>
     <message>
         <location line="-23"/>
         <source>HttpProxy</source>
         <translation>HttpProxy</translation>
     </message>
     <message>
         <location line="-2083"/>
         <source>Icon set for dock widgets</source>
-        <translation>Iconos para widget acoplados</translation>
+        <translation>Iconos para widget</translation>
     </message>
     <message>
         <location line="-48"/>
         <source>Language (requires restart)</source>
-        <translation>Lenguaje (reinicio requerido)</translation>
+        <translation>Lenguaje (requiere reinicio)</translation>
     </message>
     <message>
         <location line="-58"/>
         <source>Icon size</source>
         <translation>Tamaño de icono</translation>
     </message>
     <message>
         <location line="+1714"/>
@@ -2407,27 +2359,27 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="+132"/>
         <source>Colors for variable attributes </source>
         <translation>Colores para los atributos de variable</translation>
     </message>
     <message>
         <location line="+34"/>
         <source>Shortcuts</source>
-        <translation>Accesos directos</translation>
+        <translation>Atajos de teclado</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Select this option to prevent conflicts with readline shortcuts</source>
-        <translation type="unfinished">Seleccione esta opción para evitar conflictos con los atajos de la línea de lectura</translation>
+        <translation>Seleccione esta opción para evitar conflictos con los atajos de la línea de comandos</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Disable global shortcuts when terminal window has focus</source>
-        <translation type="unfinished">Deshabilitar accesos directos globales cuando la ventana del terminal se enfoque </translation>
+        <translation>Deshabilitar loa atajos globales cuando la ventana del terminal se enfoque </translation>
     </message>
     <message>
         <location line="+31"/>
         <location line="+28"/>
         <source>Use this set</source>
         <translation>Utilice este conjunto</translation>
     </message>
     <message>
@@ -2445,42 +2397,42 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
         <location line="-30"/>
         <location line="+37"/>
         <source>Import</source>
         <translation>Importar</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Set 2:</source>
-        <translation>Set 2:</translation>
+        <translation>Conjunto 2:</translation>
     </message>
     <message>
         <location line="+86"/>
         <source>Action</source>
         <translation>Acción</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Set 1 Default</source>
         <translation>Conjunto 1 por defecto</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set 1 Actual</source>
-        <translation type="unfinished">Conjunto 1 actual</translation>
+        <translation>Conjunto 1 actual</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Set 2 Default</source>
         <translation>Conjunto 2 por defecto</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Set 2 Actual</source>
-        <translation type="unfinished">Conjunto 2 actual</translation>
+        <translation>Conjunto 2 actual</translation>
     </message>
     <message>
         <location line="+68"/>
         <source>Socks5Proxy</source>
         <translation>&quot;Socks5Proxy&quot;</translation>
     </message>
     <message>
         <location line="-16"/>
@@ -2532,45 +2484,45 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
     <message>
         <location line="+213"/>
         <source>Difference to the default size</source>
         <translation>Diferencia con el tamaño predeterminado</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255,0,255) means default</source>
-        <translation>Color de fondo, rosa (255,0,255) significa predeterminado</translation>
+        <translation>Color de fondo. Rosa (255,0,255) significa predeterminado</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>b</source>
         <comment>short form for bold</comment>
-        <translation>b</translation>
+        <translation>n</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>i</source>
         <comment>short form for italic</comment>
-        <translation>i</translation>
+        <translation>c</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>u</source>
         <comment>short form for underlined</comment>
-        <translation>u</translation>
+        <translation>s</translation>
     </message>
     <message>
         <location line="+497"/>
         <source>Set Octave Startup Directory</source>
-        <translation>Ajuste directorio de inicio de Octave</translation>
+        <translation>Ajuste del directorio de inicio de Octave</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set File Browser Startup Directory</source>
-        <translation>Ajuste directorio de inicio del explorador de archivos</translation>
+        <translation>Ajuste del directorio de inicio del explorador de archivos</translation>
     </message>
 </context>
 <context>
     <name>setup_community_news</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+44"/>
         <source>Community News</source>
         <translation>Noticias de la comunidad</translation>
@@ -2594,17 +2546,17 @@ Asegúrese de que tiene permisos de escritura sobre\n%2\n\nOctave GUI se cerrará.</translation>
         <location line="+7"/>
         <source>&lt;html&gt;&lt;body&gt;
 &lt;p&gt;When the Octave GUI starts, it will check the Octave web site for current news and information about the Octave community.
 The check will happen at most once each day and news will only be displayed if there is something new since the last time you viewed the news.&lt;/p&gt;
 &lt;p&gt;You may also view the news by selecting the &quot;Community News&quot; item in the &quot;Help&quot; menu in the GUI, or by visiting
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;body&gt;
-&lt;p&gt;Cuando la GUI de Octave se inicia, chequeará el sitio web de Octave para obtener novedades e información acerca de la comunidad.
+&lt;p&gt;Cuando la GUI se inicia, chequeará el sitio web de Octave para obtener novedades e información acerca de la comunidad.
 Esta verificación se hará una vez cada día como máximo, y las noticias serán desplegadas únicamente si hay algo nuevo desde la última vez que se vieron.&lt;/p&gt;
 &lt;p&gt;También puede revisar las noticias seleccionando &quot;Noticias de la Comunidad&quot; en el menú de ayuda en la GUI o visitando
 &lt;a href=&quot;http://octave.org/community-news.html&quot;&gt;http://octave.org/community-news.html&lt;/a&gt;.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+26"/>
         <source>&lt;html&gt;&lt;head&gt;
@@ -2614,17 +2566,17 @@ a:link { text-decoration: underline; col
 &lt;head/&gt;&lt;body&gt;
 &lt;p&gt;Allow Octave to connect to the Octave web site when it starts to display current news and information about the Octave community.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
-&lt;p&gt;Permitir la conexión con el sitio web de Octave al arranque para desplegar novedades e información acerca de la comunidad.&lt;/p&gt;
+&lt;p&gt;Permitir la conexión con el sitio web de Octave en el arranque para desplegar novedades e información acerca de la comunidad.&lt;/p&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>shortcut_manager</name>
     <message>
         <location filename="../src/shortcut-manager.cc" line="+109"/>
         <source>New File</source>
@@ -2648,17 +2600,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Load Workspace</source>
         <translation>Cargar espacio de trabajo</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Save Workspace As</source>
-        <translation>Guardar espacio de trabajo como</translation>
+        <translation>Guardar el espacio de trabajo como</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+158"/>
         <source>Preferences</source>
         <translation>Preferencias</translation>
     </message>
     <message>
@@ -2713,37 +2665,37 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Clear Workspace</source>
         <translation>Limpiar el espacio de trabajo</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Step Over</source>
-        <translation type="unfinished">Saltar</translation>
+        <translation>Saltar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Step Into</source>
-        <translation type="unfinished">Entrar a</translation>
+        <translation>Entrar a</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Step Out</source>
-        <translation type="unfinished">Abandonar</translation>
+        <translation>Salir</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Continue</source>
         <translation>Continuar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Quit Debug Mode</source>
-        <translation>Abandonar modo de depuración</translation>
+        <translation>Abandonar el modo de depuración</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Command Window</source>
         <translation>Mostrar la ventana de comandos</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2799,22 +2751,22 @@ a:link { text-decoration: underline; col
     <message>
         <location line="-214"/>
         <source>Documentation</source>
         <translation>Documentación</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Reset Window Layout</source>
-        <translation>Reestablecer esquema de ventana</translation>
+        <translation>Restablecer esquema de ventana</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show Ondisk Documentation</source>
-        <translation type="unfinished">Mostrar la documentación del disco</translation>
+        <translation>Mostrar la documentación almacenada en disco</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Online Documentation</source>
         <translation>Mostrar la documentación en línea</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2829,17 +2781,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+1"/>
         <source>Share Code</source>
         <translation>Compartir código</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Contribute to Octave</source>
-        <translation>Colaborar en Octave</translation>
+        <translation>Colaborar con Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Octave Developer Resources</source>
         <translation>Recursos para el desarrollador de Octave</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2904,32 +2856,32 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+3"/>
         <source>Find and Replace</source>
         <translation>Buscar y reemplazar</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Word</source>
-        <translation type="unfinished">Eliminar hasta el inicio de palabra</translation>
+        <translation>Eliminar hasta el inicio de palabra</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Word</source>
-        <translation type="unfinished">Eliminar hasta el fin de palabra</translation>
+        <translation>Eliminar hasta el fin de palabra</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Line</source>
-        <translation type="unfinished">Eliminar hasta el inicio de línea</translation>
+        <translation>Eliminar hasta el inicio de línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Line</source>
-        <translation type="unfinished">Eliminar hasta el final de línea</translation>
+        <translation>Eliminar hasta el final de línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete Line</source>
         <translation>Eliminar línea</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -2944,54 +2896,54 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Duplicate Selection/Line</source>
         <translation>Duplicar selección/línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
-        <translation>Transponer línea</translation>
+        <translation>Intercambiar con línea anterior</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Completion List</source>
-        <translation type="unfinished">Lista de terminación</translation>
+        <translation>Lista de comandos coincidentes</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Comment Selection</source>
-        <translation>Comentar selección</translation>
+        <translation>Comentar la selección</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Uncomment Selection</source>
-        <translation>Descomentar selección</translation>
+        <translation>Descomentar la selección</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Uppercase Selection</source>
-        <translation>Aplicar mayúsculas a selección</translation>
+        <translation>Convertir la selección a mayúsculas</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Lowercase Selection</source>
-        <translation>Aplicar minúsculas a selección</translation>
+        <translation>Convertir la selección a minúsculas</translation>
     </message>
     <message>
         <location line="+4"/>
         <location line="+5"/>
         <source>Indent Selection</source>
-        <translation>Aplicar sangría a selección</translation>
+        <translation>Aplicar sangría a la selección</translation>
     </message>
     <message>
         <location line="-3"/>
         <location line="+5"/>
         <source>Unindent Selection</source>
-        <translation>Reducir sangría a selección</translation>
+        <translation>Reducir sangría a la selección</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Convert Line Ednings to Windows</source>
         <translation>Convertir los finales de línea a Windows</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -3001,47 +2953,47 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Convert Line Ednings to Mac</source>
         <translation>Convertir los finales de línea a Mac</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Goto Line</source>
-        <translation>Ir a línea</translation>
+        <translation>Ir a la línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move to Matching Brace</source>
-        <translation>Mover a la llave correspondiente</translation>
+        <translation>Mover el paréntesis/corchete/llave correspondiente</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Select to Matching Brace</source>
-        <translation>Seleccionar a la llave correspondiente</translation>
+        <translation>Seleccionar el paréntesis/corchete/llave correspondiente</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Toggle Bookmark</source>
-        <translation type="unfinished">Alternar marcador (Bookmark)</translation>
+        <translation>Alternar marcador</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Next Bookmark</source>
-        <translation type="unfinished">Marcador siguiente (Sig. Bookmark)</translation>
+        <translation>Siguiente marcador</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Previous Bookmark</source>
-        <translation type="unfinished">Marcador anterior (Ant. Bookmark)</translation>
+        <translation>Marcador anterior</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Remove All Bookmark</source>
-        <translation type="unfinished">Eliminar todos los marcadores (Bookmarks)</translation>
+        <translation>Eliminar todos los marcadores</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Styles Preferences</source>
         <translation>Preferencias de estilos</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -3051,37 +3003,37 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Show White Spaces</source>
         <translation>Mostrar espacios en blanco</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Line Endings</source>
-        <translation>Mostrar fin de líneas</translation>
+        <translation>Mostrar fin de línea</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Show Indentation Guides</source>
         <translation>Mostrar guías de sangría</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Show Long Line Marker</source>
-        <translation>Mostrar marcador largo de línea</translation>
+        <translation>Mostrar marcador de longitud de línea</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom In</source>
-        <translation type="unfinished">Aumentar zoom</translation>
+        <translation>Aumentar</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Zoom Out</source>
-        <translation type="unfinished">Disminuir zoom</translation>
+        <translation>Disminuir</translation>
     </message>
     <message>
         <location line="+2"/>
         <location line="+3"/>
         <source>Zoom Normal</source>
         <translation>Zoom normal</translation>
     </message>
     <message>
@@ -3092,17 +3044,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Next Breakpoint</source>
         <translation>Siguiente punto de interrupción</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Previous Breakpoint</source>
-        <translation>Punto de interrupción previo</translation>
+        <translation>Punto de interrupción anterior</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Remove All Breakpoints</source>
         <translation>Eliminar todos los puntos de interrupción</translation>
     </message>
     <message>
         <location line="+4"/>
@@ -3122,17 +3074,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+2"/>
         <source>Document on Keyword</source>
         <translation>Documentación de palabras clave</translation>
     </message>
     <message>
         <location line="+42"/>
         <source>Main</source>
-        <translation type="unfinished">Principal</translation>
+        <translation>Principal</translation>
     </message>
     <message>
         <location line="+3"/>
         <location line="+23"/>
         <source>File</source>
         <translation>Archivo</translation>
     </message>
     <message>
@@ -3171,112 +3123,112 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+4"/>
         <source>Run</source>
         <translation>Ejecutar</translation>
     </message>
     <message>
         <location line="+124"/>
         <source>Enter new Shortcut for Set %1</source>
-        <translation type="unfinished">Introduzca nuevo acceso directo para conjunto %1</translation>
+        <translation>Introduzca nuevo atajo para conjunto %1</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Apply the desired shortcut or click on the right button to reset the shortcut to its default.</source>
-        <translation type="unfinished">Aplicar acceso directo que desee o haga clic en el botón derecho para restablecer el acceso directo a su valor predeterminado.</translation>
+        <translation>Aplique el atajo que desee o haga clic en el botón derecho para restablecer el atajo a su valor predeterminado.</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Enter shortcut directly by performing it</source>
-        <translation type="unfinished">Introduzca directamente un acceso directo</translation>
+        <translation>Introduzca directamente un atajo de teclado</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Actual shortcut</source>
-        <translation type="unfinished">Acceso directo actual</translation>
+        <translation>Atajo de teclado actual</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Default shortcut</source>
-        <translation type="unfinished">Acceso directo por defecto</translation>
+        <translation>Atajo de teclado por defecto</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Set to default</source>
-        <translation type="unfinished">Establecer como por defecto</translation>
+        <translation>Establecer como por defecto</translation>
     </message>
     <message>
         <location line="+53"/>
         <source>Double Shortcut</source>
-        <translation type="unfinished">Acceso directo doble</translation>
+        <translation>Atajo de teclado doble</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The chosen shortcut
   &quot;%1&quot;
 is already used for the action
   &quot;%2&quot;.
 Do you want to use the shortcut anyhow removing it from the previous action?</source>
-        <translation type="unfinished">El acceso directo seleccionado
+        <translation>El atajo de teclado seleccionado
   &quot;%1&quot;
 ya se utiliza para la acción
   &quot;%2&quot;.
-¿Desea utilizar el acceso directo de cualquier modo removiendo la acción previamente asignada?</translation>
+¿Desea utilizar el atajo de cualquier modo suprimiendo la acción previamente asignada?</translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Import shortcut set %1 from file ...</source>
-        <translation type="unfinished">Importar conjunto de acceso directo %1 del archivo ...</translation>
+        <translation>Importar conjunto de atajos de teclado %1 del archivo ...</translation>
     </message>
     <message>
         <location line="+1"/>
         <location line="+6"/>
         <source>Octave Shortcut Files (*.osc);;All Files (*)</source>
-        <translation type="unfinished">Archivo de accesos directos de Octave (*.osc);;Todos los archivos (*)</translation>
+        <translation>Archivo de atajos de teclado de Octave (*.osc);;Todos los archivos (*)</translation>
     </message>
     <message>
         <location line="-1"/>
         <source>Export shortcut set %1 into file ...</source>
-        <translation type="unfinished">Exportar conjunto de accesos directos %1 al archivo ...</translation>
+        <translation>Exportar conjunto de atajos de teclado %1 al archivo ...</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Failed to open %1 as octave shortcut file</source>
-        <translation type="unfinished">No se ha podido abrir %1 como archivo de accesos directos de Octave</translation>
+        <translation>No se ha podido abrir %1 como archivo de atajos de teclado de Octave</translation>
     </message>
 </context>
 <context>
     <name>terminal_dock_widget</name>
     <message>
         <location filename="../src/terminal-dock-widget.cc" line="+38"/>
         <source>Command Window</source>
         <translation>Ventana de comandos</translation>
     </message>
 </context>
 <context>
     <name>webinfo</name>
     <message>
         <location filename="../src/qtinfo/webinfo.cc" line="+83"/>
         <source>Type here and press &apos;Return&apos; to search</source>
-        <translation>Escriba aquí y pulse la tecla &apos;Retorno&apos; para buscar</translation>
+        <translation>Escriba aquí y pulse la tecla &apos;enter&apos; para buscar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Global search</source>
         <translation>Búsqueda global</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Error</source>
         <translation>Error</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The info file&lt;p&gt;%1&lt;p&gt;or compressed versions do not exist</source>
-        <translation type="unfinished">El archivo de información &lt;p&gt;%1&lt;p&gt; o la versión comprimida no existen</translation>
+        <translation>El archivo de información &lt;p&gt;%1&lt;p&gt; o la versión comprimida no existen</translation>
     </message>
 </context>
 <context>
     <name>welcome_wizard</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+135"/>
         <source>Welcome to GNU Octave</source>
         <translation>Bienvenido a GNU Octave</translation>
